demo/runners/support/agent.py

```diff
@@ -38,6 +36,8 @@
 
 AGENT_ENDPOINT = os.getenv("AGENT_ENDPOINT")
 
+PUBLIC_TAILS_URL = os.getenv("PUBLIC_TAILS_URL")
```

Comment:
Remove references to PUBLIC_TAILS_URL in agent.py?  Note that this code will never get reached due to the validation check for `if args.revocation and not args.tails_server_base_url` in faber.py

OR move use of this env var into faber.py?